src/lib/seam/connect/route-types.ts

```diff
@@ -3583,7 +3583,6 @@ export interface Routes {
         workspace_id: string
         name: string
         is_sandbox: boolean
-        connect_partner_name: string | null
```

Comment:
@codetheweb Not sure why this prop was removed, if you look at the upstream PR, workspace was not changed.

Reply:
I believe this PR included changes from a few upstream PRs. There were 4 open PRs on this repo today, I merged this one (the most recent) and closed the others.